index.js

```diff
@@ -25,5 +25,24 @@ export {
   text,
   attr,
 } from './src/virtual_elements';
-export { attributes } from './src/attributes';
+
+
+import {
```

Comment:
I think rather than having logic like this in the index.js, I would prefer to have separate modules for these and just directly export them.